src/SonarLint.Test.props

```diff
@@ -11,7 +11,17 @@
     <SonarQubeTestProject>true</SonarQubeTestProject>
 
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)TestProjects.ruleset</CodeAnalysisRuleSet>
-
   </PropertyGroup>
+
+  <ItemGroup Label="Test framework">
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
+    <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
+    <PackageReference Include="Moq" Version="4.18.2" />
+    
+    <!-- ??? Needs to use the same version as the MS.VS.SDK.TestFramework ???-->
```

Comment:
I'm not sure where I copied this comment from or if it's still accurate, hence the `???` as a reminder next time we change it.